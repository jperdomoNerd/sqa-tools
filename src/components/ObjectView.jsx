const ObjectView = () => {
  return (
    <div className="object-view"></div>
  );
}

export default ObjectView;