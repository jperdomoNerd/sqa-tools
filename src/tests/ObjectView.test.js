import renderer from 'react-test-renderer';
import ObjectView from '../components/ObjectView';

describe('ObjectView Component', () => {
  
  it('<OjbectView /> renders correctly', () => {
    const tree = renderer.create(<ObjectView />).toJSON();
    expect(tree).toMatchSnapshot();
  });

});
