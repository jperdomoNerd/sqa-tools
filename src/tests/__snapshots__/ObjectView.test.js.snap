// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ObjectView Component <OjbectView /> renders correctly 1`] = `
<div
  className="object-view"
/>
`;
